src/server/connection.rs

```diff
@@ -1761,6 +1821,22 @@ impl Connection {
                 }
             }
         } else if self.authorized {
+            #[cfg(target_os = "windows")]
+            if !self.checked_multiple_session {
+                self.checked_multiple_session = true;
+                if crate::platform::is_installed()
+                    && crate::platform::is_share_rdp()
+                    && !*CONN_COUNT.lock().unwrap() > 1
```

Comment:
when the second client connects, the session chosen dialog still pops up, it's not a big problem, but if the user choose a session different from the current one, the two clients will reconnect in a loop.